pkg/report/table.go

```diff
@@ -50,6 +50,9 @@ func (tw TableWriter) write(result types.Result) {
 
 	target := result.Target
 	if result.Class == types.ClassSecret {
+		if len(result.Secrets) == 0 {
```

Comment:
Correct me if I'm wrong, but when the Class is `secret`, all the other results will be always empty.
@knqyf263

Reply:
Yes, it should be.